ycms/cms/templates/timeline/timeline.html

```diff
@@ -7,25 +7,30 @@
 {% block content %}
     <div class="flex justify-center items-center">
         <div class="container mx-auto">
-            <div class="col-span-full rounded-lg border border-gray-300 bg-white shadow-md flex flex-nowrap h-full items-center justify-between p-4 mb-4 hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700">
+            <div class="flex flex-col xl:flex-row rounded-lg border border-gray-300 bg-white shadow-md flex flex-nowrap h-full items-center justify-between p-4 mb-4 hover:bg-gray-100 dark:bg-gray-800 dark:border-gray-700 dark:hover:bg-gray-700">
                 {% include "ward/ward_selection_form.html" %}
                 <div id="change-counter"
-                     class="hidden flex items-center gap-2 dark:text-white">
-                    <span>0</span> {% translate "unsaved bed assignment changes" %}
-                    <form enctype="multipart/form-data"
-                          method="post"
-                          id="timeline-changes-form"
-                          class="flex items-center">
-                        {% csrf_token %}
-                        <input type="hidden" name="timeline_changes" id="timeline-changes" />
-                        <button class="text-white bg-green-500 hover:bg-green-700 focus:ring-4 focus:outline-none focus:ring-green-300 rounded inline-flex items-center px-5 py-2.5 text-center">
-                            {% translate "Save" %}
+                     class="hidden flex flex-col lg:flex-row items-center gap-2 dark:text-white">
+                    <div class="flex flex-row gap-2 w-full">
```

Comment:
```suggestion
                    <div class="shrink-0">
```
No difference on mobile, but on Desktop:

Without: 
![image](https://github.com/charludo/ycms/assets/47758554/20352085-a555-4e84-b46f-f2fb588dc09f)

With: 
![image](https://github.com/charludo/ycms/assets/47758554/25a6e8cd-a77a-4872-8a55-2c4c4e65316e)